azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/models/ParticipantTimestampInfoModel.kt

```diff
@@ -9,4 +9,26 @@ import org.threeten.bp.OffsetDateTime
 internal data class ParticipantTimestampInfoModel(
     val userIdentifier: CommunicationIdentifier,
     val receivedOn: OffsetDateTime,
-) : BaseInfoModel
+) : BaseInfoModel {
+    companion object {
+        fun fromMessageInfoModel(infoModel: MessageInfoModel): ParticipantTimestampInfoModel =
```

Comment:
would it be possible to move this logic outside?

Reply:
@iaulakh what destination do you suggest for that block of code

Reply:
We can create helper class Albert.

Reply:
done